.classpath

```diff
@@ -157,15 +157,19 @@
 			<attribute name="m2e-apt" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
 	<classpathentry kind="src" output="target/test-classes" path="target/generated-test-sources/test-annotations">
 		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-			<attribute name="test" value="true"/>
 			<attribute name="optional" value="true"/>
 			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="ignore_optional_problems" value="true"/>
 			<attribute name="m2e-apt" value="true"/>
+			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
```

Comment:
I'm perhaps overly cautious about changes to the Eclipse files, because they seem obscure to me and we've had obscure problems pop up in the past.  Is this just a rearrangement?  Or does this change the behavior of whatever `JRE_CONTAINER` does?

Reply:
Visual Studio Code and Eclipse both use [Eclipse JDT Language Server](https://github.com/eclipse/eclipse.jdt.ls) for Java processing. Visual Studio Code did this automatically, and I can't see any difference in Eclipse's behavior, so I am assuming it is correct.